### PR TITLE
Fixed some errors in example conf files. Added a logFileMaxBackupFile…

### PR DESCRIPTION
**Pull request by larspehrsson**
…s to allow for more than 1 backup

### CODE REVIEW

File: lib/logging/logging.go [hunk 1]
```diff
@@ -135,13 +137,28 @@ func (rf *rollingFile) open() error {
...
+		if i == 1 {
+			olderFile = rf.file.Name() // Original file
+		} else {
+			olderFile = fmt.Sprintf("%s.%d", rf.file.Name(), i-1)
+		}
+		newerFile = fmt.Sprintf("%s.%d", rf.file.Name(), i)
+
+		// If the older file exists, rename it to the next higher numbered file
+		if _, err := os.Stat(olderFile); err == nil {
+			os.Rename(olderFile, newerFile)
```
Comment (hiroxy) [inline lib/logging/logging.go:157]: An error returned by `os.Rename` should be verified. Then, calling `os.Stat` would be unnecessary.
Probably,
```go
switch err := os.Rename(olderFile, newerFile); {
case err == nil, errors.Is(err, os.ErrNotExist):
    continue
case err != nil:
    // error handling
}
```

File: lib/logging/logging.go [hunk 2]
```diff
@@ -135,13 +137,28 @@ func (rf *rollingFile) open() error {
 }
 
 func (rf *rollingFile) roll() error {
-	// FUTURE: Support more than one roll.
+	// Flush the current buffer
```
Comment (hiroxy) [inline lib/logging/logging.go:140]: NIT: unnecessary comment since we know `Flush` flushes the buffer.

File: lib/logging/logging_test.go
Comment (hiroxy) [inline lib/logging/logging_test.go:1]: It should be tested that the logs are kept for the specified generation.

**Review by hiroxy — CHANGES_REQUESTED** [review]

File: lib/logging/logging.go [hunk 3]
```diff
@@ -152,12 +152,14 @@ func (rf *rollingFile) roll() error {
 		}
 		renameTo = fmt.Sprintf("%s.%d", rf.file.Name(), i)
 
-		switch err := os.Rename(renameFrom, renameTo); {
-		case err == nil, errors.Is(err, os.ErrNotExist):
+		if err := os.Rename(renameFrom, renameTo); err == nil || errors.Is(err, os.ErrNotExist) {
+			// Continue if no error or if the error is 'file not found'
 			continue
-		case err != nil:
+		} else {
```
Comment (hiroxy) [inline lib/logging/logging.go:158]: NIT: No else statement is required.

**Review by hiroxy — APPROVED** [review]

File: lib/logging/logging.go (hunk 3 above)
Comment (larspehrsson) [reply lib/logging/logging.go:158]: I could remove it, but the err is only scoped within the if then else.

**Pull request merged**